release-notes/6.0/preview/6.0.0-preview.4-runtime-deps.json

```diff
@@ -0,0 +1,1035 @@
+{
+  "dotnetReleaseVersion": "6.0.0-preview.4",
+  "dependencyUsages": {
+    "default": "Used by default or for canonical scenarios",
+    "diagnostics": "Used for diagnostic scenarios such as tracing or debugging",
+    "graphics": "Used for scenarios involving graphics/drawing/fonts",
```

Comment:
"graphics" does not apper to be used anywhere

Reply:
Removed